force-app/main/default/classes/Chainable.cls

```diff
@@ -1,5 +1,8 @@
 public abstract class Chainable {
 
+	@TestVisible
```

Comment:
Please check indentation accross all files. Looks like you are using tabs or other spaces. This TestVisible needs to be aligned with the private below